Framework/DataHandling/src/LoadEventNexus.cpp

```diff
@@ -855,13 +857,28 @@ void LoadEventNexus::loadEvents(API::Progress *const prog,
       static_cast<double>(std::numeric_limits<uint32_t>::max()) * 0.1;
   longest_tof = 0.;
 
-  bool precount = getProperty("Precount");
-  int chunk = getProperty("ChunkNumber");
-  int totalChunks = getProperty("TotalChunks");
-  DefaultEventLoader::load(this, *m_ws, haveWeights, event_id_is_spec,
-                           bankNames, periodLog->valuesAsVector(), classType,
-                           bankNumEvents, oldNeXusFileNames, precount, chunk,
-                           totalChunks);
+  bool loaded{false};
+  if (canUseParallelLoader(haveWeights, oldNeXusFileNames, classType)) {
+    auto ws = m_ws->getSingleHeldWorkspace();
+    m_file->close();
+    try {
+      ParallelEventLoader::load(*ws, m_filename, m_top_entry_name, bankNames);
+      loaded = true;
+      shortest_tof = 0.0;
+      longest_tof = 1e10;
+    } catch (const std::runtime_error &) {
+    }
```

Comment:
Yes, it will fall back to the default loader. In fact I will need to make more changes such that the parallel loader is never used by default, unless enabled explicitly.